hadoop-ozone/dev-support/checks/acceptance.sh

```diff
@@ -41,6 +41,7 @@ export OZONE_ACCEPTANCE_SUITE
 cd "$DIST_DIR/compose" || exit 1
 ./test-all.sh 2>&1 | tee "${REPORT_DIR}/output.log"
 RES=$?
-cp result/* "$REPORT_DIR/"
+cp -rv result/* "$REPORT_DIR/"
 cp "$REPORT_DIR/log.html" "$REPORT_DIR/summary.html"
```

Comment:
Not related to the PR, but I wonder: why do we copy `log.html` to `summary.html`? Can we do a `mv`? Does some other script expect both `log.html` and `summary.html`?